transform-service/latest/install/index.md

```diff
@@ -310,7 +351,7 @@ from the left column that corresponds to the required Content Services version y
 
     As an alternative, you can also start the containers in the background by running `docker-compose up -d`.
 
-4. Wait for the logs to show messages:
+5. Wait for the logs to show messages:
 
     ```bash
     alfresco_1 | 2020-07-06 11:50:46,808  WARN ... The Alfresco Content Services license will expire in 2 days.
```

Comment:
I suggest updating the dates (lines 357-359: `2020-07-06`) to coincide with the latest release, or at least be in 2022.